src/lib/Legislator.svelte

```diff
@@ -130,8 +130,11 @@
   }
 
```

Comment:
I'm open to it, feel free to open as a PR? I think the bullets are kind of nice because all of the letters are capitalized so it's hard to tell where one committee name starts and another ends sometimes.